lldb/docs/use/map.rst

```diff
@@ -1126,6 +1126,20 @@ Save binary memory data starting at ``0x1000`` and ending at ``0x2000`` to a fil
   (lldb) memory read --outfile /tmp/mem.bin --binary 0x1000 0x2000
   (lldb) me r -o /tmp/mem.bin -b 0x1000 0x2000
 
+
+Print information about memory regions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Remove the extra `~`.

Reply:
I don't think it makes an actual difference for titles but might as well make it neat. Unlike other places in RST where stray characters change the entire document seemingly randomly :)